spec/controllers/send_application_controller_spec.rb

```diff
@@ -34,6 +34,15 @@
           current_app.reload.email
         }.from("test@example.com").to("new_email@example.com")
       end
+
+      it "creates a SendApplicationJob" do
```

Comment:
It's fine, I believe. All you want to know is that the job is created by the controller, so this is clear to me.

Is the job itself in another PR coming down the pipe?